platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/event/preconsensus/PreConsensusEventFileManagerTests.java

```diff
@@ -574,7 +574,7 @@ void incrementalPruningByGenerationTest() throws IOException {
 
             minimumGeneration = random.nextLong(minimumGeneration, maximumGeneration + 1);
             maximumGeneration =
-                    Math.max(maximumGeneration, random.nextLong(minimumGeneration, minimumGeneration + maxDelta));
+                    Math.max(maximumGeneration + 1, random.nextLong(minimumGeneration, minimumGeneration + maxDelta));
```

Comment:
This exact pattern exists elsewhere in these tests. Are you intentionally modifying it only for this single case?

Reply:
This particular test is sensitive to the scenario where the maximum generations of two concurrent files perfectly align. This is because I am pruning by generation, and sometimes it will prune multiple files in a row if these numbers line up.

It's much simpler in this test to avoid the scenario where multiple generations line up like this. Other tests do not make assertions that care about this scenario.